case_different_celltypes/Input/tube.in

```diff
@@ -0,0 +1,30 @@
+0.44 !0.04              ! alpha_Ewld !changed for big tube
+1.E-3                   ! eps_Ewld
+8                       ! PBspln_Ewld
+
+3                       ! nCellTypes
+1 1 3                   ! viscRat (Sickle Cells experience ~3 to 4x viscosity as a healthy RBC) Reference: https://doi.org/10.1016/j.actbio.2012.07.011
```

Comment:
make this 1. unclear if the simulation is stable for viscRat /= 1.

Reply:
Will change back to all 1s in rev3